packages/core/src/middleware/koa-root-proxy.ts

```diff
@@ -0,0 +1,21 @@
+import { MiddlewareType } from 'koa';
+import { IRouterParamContext } from 'koa-router';
+
+export default function koaRootProxy<
+  StateT,
+  ContextT extends IRouterParamContext,
+  ResponseBodyT
+>(): MiddlewareType<StateT, ContextT, ResponseBodyT> {
+  return async (ctx, next) => {
+    const requestPath = ctx.request.path;
+
+    // Empty path return 404
+    if (requestPath === '/') {
+      ctx.throw(404);
+
+      return;
```

Comment:
Maybe we can redirect to `/console`?

Reply:
There is another short discussion in the previous PR. Keep 404 for now.

Reply:
ok